tests/config/bindings/tests.yml

```diff
@@ -1,13 +1,18 @@
-# the config map is passed in a metadata map to the Invoke request for output bindings
-componentType: output-binding
+componentType: bindings
 components:
   - component: redis
     operations: ["create", "operations"]
     config:
       key: $((uuid))
+      readBindingTimeout: 60
   - component: azure.blobstorage
     operations: ["create", "operations", "get"]
     config:
       blobName: $((uuid))
+      readBindingTimeout: 60
   - component: azure.storagequeues
+    operations: ["create", "operations", "read"]
+  - component: kafka
     operations: ["create", "operations"]
+    config:
+      readBindingTimeout: 60
```

Comment:
can you distinguish between `input_` metadata and `output_` metadata? This way all the `output_` metadata alone can be passed along to the InvokeRequest?

Reply:
Done.